students/hyeongtaek/westagram/westagram/settings.py

```diff
@@ -0,0 +1,154 @@
+"""
+Django settings for westagram project.
+
+Generated by 'django-admin startproject' using Django 4.0.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+
+from pathlib     import Path #기존에 settings.py 에 있는 코드
```

Comment:
다음과 같은 불필요한 주석은 지워주세요~
```suggestion
from pathlib     import Path
```